### PR TITLE
Add Modal Component

### PR DESCRIPTION
**Pull request by sellalone**
Add Modal Component #1

### CODE REVIEW

File: snippets/Modal.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+`keydownHandler`, is a method which handles all the Keyboard events.
+
+The Modal Component has following attributes:
+
+- `isVisible`, Boolean describing if the modal should be shown or not.(Required)
+- `title`, String describing title of Modal.(Required)
+- `content`, JSX elements to be rendered in modal body.(Required)
+- `footer`, JSX elements to be rendered in modal footer. (optional)
+- `onClose`,  Function that will be called when Modal needs to be closed like when escape key is pressed.(Required)
+
```
Comment (Chalarangelo) [inline snippets/Modal.md]: Please reduce the whitespace to one line between sections.

File: snippets/Modal.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  <Modal 
+    isVisible={false}
+    title= "Modal Title"
+    content={<p>The content goes here</p>}
+    footer={<button>Cancel</button>}        
+    onClose ={()=> } //Add action to hide modal
+/>,
+  document.getElementById('root')
+);
+```
```
Comment (Chalarangelo) [inline snippets/Modal.md]: I do not see any `tags` or `expertise`, please add these according to the template.

**Comment by sellalone** [comment]
Hey Chalarangelo, I have updated the PR but to be honest im not that good at documentation so i need your help for that.
Thanks

File: snippets/Modal.md [hunk 3]
```diff
@@ -123,14 +118,26 @@ function Modal(props){
...
+        onClose ={()=> setModal(false)}
+      />
+    </React.Fragment>
+  )
+}
+
+ReactDOM.render( <App/>, document.getElementById('root'));
 ```
+
+<!-- tags: modal -->
```
Comment (Chalarangelo) [inline snippets/Modal.md]: Please use existing tags, mainly. Using the component's name as a tag is not really all that helpful.

**Comment by sellalone** [comment]
Hey Chalarangelo,
Can you please what is remaining?

**Comment by sellalone** [comment]
Thanks bro

**Pull request merged**